ts/features/itwallet/presentation/details/screens/ItwPresentationCredentialDetailScreen.tsx

```diff
@@ -70,33 +74,81 @@ type Props = IOStackNavigationRouteProps<
 /**
  * Component that renders the credential detail screen.
  */
-export const ItwPresentationCredentialDetailScreen = ({ route }: Props) => {
+export const ItwPresentationCredentialDetailScreen = ({
+  navigation,
```

Comment:
You can obtain the `navigation` object using `const navigation = useIONavigation();`
We barely get the navigation object from the props.